internal/mountinfo/mountinfo.go

```diff
@@ -0,0 +1,206 @@
+package mountinfo
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	sglog "github.com/sourcegraph/log"
+	"golang.org/x/sys/unix"
+)
+
+// DefaultSysMountPoint is the common mount point for the sysfs pseudo-filesystem.
+const DefaultSysMountPoint = "/sys"
+
+// MustRegisterNewMountPointInfoMetric registers a Prometheus metric named "mount_point_info" that
```

Comment:
I think this function doc is quite wordy...I'd appreciate suggestions on how to improve the clarity here.